### ISSUE TITLE
plz help me. Just reinstall the nmm  and having this

### ISSUE THREAD

**Issue by giang6996** [Nexus-Mods/Nexus-Mod-Manager#671]
_Please give as much information as you can, replace/remove italicized text._

**Describe the bug** 
_A short description of what the problem is_

[
[TraceLog20181211133041.txt](https://github.com/Nexus-Mods/Nexus-Mod-Manager/files/2666312/TraceLog20181211133041.txt)
[TraceLog20181211133157.txt](https://github.com/Nexus-Mods/Nexus-Mod-Manager/files/2666313/TraceLog20181211133157.txt)

](url)

**Environment**
- NMM version: _0.1.0_
- Mod(s) used: _<link to mod(s) on Nexus>_

_Please remember to attach any trace logs you get in connection to this problem. Without these it will often be very hard to track down the bug._

**To Reproduce**
 _How to reproduce the problem_

**Additional information** 
_Anything that could help track down this problem, other software used, screenshots, etc._

**Comment by benjamindoron** [comment]
The tracelog says that the error is due to the presence of "illegal characters." The only files that match that description, by my estimation are "CMPA +100.esp," "CMPA +150.esp," etc.  I'd suggest removing those files and seeing if NMM works then.

On a separate matter, you seem to have some duplicate/redundant mods.
You have every version of "Carry More with Power Armor" - "CMPA - +*.esp" - installed.
You have Scrap Everything + all DLC addons, but also have the Ultimate Edition installed, which from memory, includes all of them.
You have a duplicated copy of "SkibsWeapons-REDUX.esp." I recommend that you remove it.

Also, I can't be sure, but you seem to be missing some files in "E:\Nexus Mod Manager\GameModes." They'd only be files relating to fomod installs, and I might be wrong, but you should check it out.

Hope this helps.

**Comment by squid-box** [comment]
I agree with bbdoron770, and as an additional bonus the actual crash is fixed in 0.65.10 - Update and see if it works better. That could also fix the numerous missing file warnings present in your tracelog.

**Comment by benjamindoron** [comment]
Sorry, changed my username (realised that it's awkward now, but I've wanted to do it for a while. I suppose awkwardness, in general, couldn't have been helped).
Yeah, I thought something was off about the plus symbol being an illegal character. The first things I looked for were extra/wrong slashes or back-slashes because I couldn't see any non-Unicode filenames.
It's immaterial though. I'm glad to see that the problem was resolved with a recent commit. Thanks.
